### PR TITLE
Scatter Vis Speed Improvements

### PR DESCRIPTION
**Pull request by dvzacharycutler**
Closes https://github.com/datavisyn/reprovisyn/issues/746

### Developer Checklist (Definition of Done)

**Issue**

- [x] All acceptance criteria from the issue are met
- [x] Tested in latest Chrome/Firefox

**UI/UX/Vis**

- [x] Requires UI/UX/Vis review
  - [ ] Reviewer(s) are notified (_tag assignees_)
  - [ ] Review has occurred (_link to notes_)
  - [ ] Feedback is included in this PR
  - [ ] Reviewer(s) approve of concept and design

**Code**

- [x] Branch is up-to-date with the branch to be merged with, i.e., develop
- [x] Code is cleaned up and formatted
- [ ] Unit tests are written (frontend/backend if applicable)
- [x] Integration tests are written (if applicable)

**PR**

- [x] Descriptive title for this pull request is provided (will be used for release notes later)
- [x] Reviewer and assignees are defined
- [x] Add type label (e.g., *bug*, *feature*) to this pull request
- [ ] Add release label (e.g., `release: minor`) to this PR following [semver](https://semver.org/)
- [x] The PR is connected to the corresponding issue (via `Closes #...`)
- [x] [Summary of changes](#summary-of-changes) is written

### Summary of changes

- Fixing a series of small bugs in the above issue, as well as making some changes to the scatterplot to make selection faster.
- Remove strip plot

### Additional notes for the reviewer(s)

-  
Thanks for creating this pull request 🤗

### CODE REVIEW

**Comment by dvzacharycutler** [comment]
Tested and discussed this PR with @thinkh  today. It seems some problems were introduced with the most recent lineup version which slowed down our animations. Ideally, we would unmount lineup during the animation and remount it afterwards. But the current way we use lineup does not allow this, as we would lose the internal state. After making a better lineup component this will potentially be fixed. 

Also, There were some recalculate styles being caused by the utils sidebar being added by the transition, which slowed things down drastically. Just keeping the sidebar at all times, for example, increased the animation performance by a lot. Should look into why this happens. 

There was also a long recalculate style being caused by react-select which were rendered on the datasets page. Not sure why it was caused, but unmounting the tabs when they arent displayed fixed the problem, which should happen anyways.

**Comment by oltionchampari** [comment]
Two issues that I found. Posting the screenshots below

1. In the combined view, the columns do not update in the column chooser, causing the below error.

Screenshot from 2023-02-02 11-16-51

2. When zooming in, there is a flickering in the scatter plot

gr

**Comment by thinkh** [comment]
@oltionchampari Thanks for testing this in Ordino. I think making this compatible with the old stack exceeds the value since the vis feature is currently only available for testing in dev instances. As long as it works correct with our modern React + Mantine stack we are good to go.

File: src/vis/interfaces.ts [hunk 1]
```diff
@@ -89,11 +86,11 @@ export interface IViolinConfig {
...
-export interface IStripConfig {
-  type: ESupportedPlotlyVis.STRIP;
-  numColumnsSelected: ColumnInfo[];
-  catColumnsSelected: ColumnInfo[];
-}
+// export interface IStripConfig {
+//   type: ESupportedPlotlyVis.STRIP;
+//   numColumnsSelected: ColumnInfo[];
+//   catColumnsSelected: ColumnInfo[];
+// }
```
Comment (thinkh) [inline src/vis/interfaces.ts:93]: I think we can remove this since we removed the whole strip plot.

```suggestion
```

**Review by thinkh — APPROVED** [review]
Thanks for the improvements. Looks good to me!

**Review by thinkh — APPROVED** [review]
Thanks for the improvements. It looks good to me.

**Pull request merged**